CONTRIBUTING.md

```diff
@@ -86,7 +92,8 @@ We don't expect that every contributor is ready to write Nightwatch tests like
 these, but it is surely appreciated. If you are submit bug fix, please do
 include a test that reproduces the issue.
 2. **Make sure existing tests pass.** Run `npm test` to run the tests, including
-the code linters.
+the code linters. Note that tests require access to the /test folder in order to
```

Comment:
I think we can remove this reference to running as administrator on Windows as long as Windows Subsystem for Linux is used, correct?

Reply:
Yep, good point - I left it originally in case someone decided to ignore the WSL advice, but honestly that's probably something that just shouldn't be supported anyway. Removed it in the latest commit.